src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -1199,6 +1199,15 @@ static void window_cheats_invalidate(rct_window* w)
     // Current weather
     window_cheats_misc_widgets[WIDX_WEATHER].text = WeatherTypes[gClimateCurrent.Weather];
     // Staff speed
+    _selectedStaffSpeed = gCheatCurrentStaffSpeed;
+    switch (gCheatCurrentStaffSpeed)
+    {
+        case CHEATS_STAFF_NORMAL_SPEED:
+            _selectedStaffSpeed = 1;
+            break;
+        case CHEATS_STAFF_FAST_SPEED:
+            _selectedStaffSpeed = 2;
```

Comment:
Doesn't look correct, you have just assigned to this variable.

Reply:
This is part of your source code, i'm just repeat them with needed variables: https://github.com/OpenRCT2/OpenRCT2/blob/de83a0b40b360e139e0d723df616ef8aa680ce4e/src/openrct2-ui/windows/Cheats.cpp#L761-L769

Reply:
These snippets don't look equivalent to me, but I'll check it later.